mappings/net/minecraft/structure/Structure.mapping

```diff
@@ -1,5 +1,5 @@
 CLASS net/minecraft/class_3499 net/minecraft/structure/Structure
-	FIELD field_15586 blocks Ljava/util/List;
+	FIELD field_15586 blockInfoLists Ljava/util/List;
```

Comment:
Is this like `List<List<...>>`?

Reply:
Effectively yes, but it is `List<PalettedBlockInfoList>`, where `PalettedBlockInfoList` contains a `List<StructureBlockInfo>`, so it's like `List<List<StructureBlockInfo>>`